packages/plugin-core/package.json

```diff
@@ -26,7 +26,8 @@
   ],
   "main": "./out/src/extension.js",
   "extensionPack": [
-    "dendron.dendron-markdown-preview-enhanced"
+    "dendron.dendron-markdown-preview-enhanced",
+    "mushan.vscode-paste-image"
```

Comment:
we actually have a fork of this because the original extension isn't published in vscodium registry. change to dendron.dendron-paste-image

Reply:
Interesting - let me change.